### PR TITLE
Prepare inputs for 3 systems: FAAH(misc), BACE prospective (janssen), BACE p3 Arg368in (Janssen)

### PR DESCRIPTION
**Pull request by nupurbansal**
## Input submission checklist

* [Done ] created a directory for each system based on the [template directory][templates]
* [ Done] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [Done ] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [Done ] added a PDB file with the protein named `protein.pdb`
* [ N/A] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [Done ] did not re-protonate the residues, but left protonation state of the protein unchanged
* [Done ] kept cystallographic waters and metals in the PDB file
* [Done ] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [Done ] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Licensing agreement
* [ Yes] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/input_structures/prepared_structures/janssen_bace/bace_ciordia_prospective/bace_ciordia_prospective_ligands.sdf
[the remark this answers is not in the document]
Comment (nupurbansal) [reply industry_benchmarks/input_structures/prepared_structures/janssen_bace/bace_ciordia_prospective/bace_ciordia_prospective_ligands.sdf:1]: Done

File: industry_benchmarks/input_structures/prepared_structures/janssen_bace/bace_ciordia_prospective/bace_ciordia_prospective_protein.pdb
[the remark this answers is not in the document]
Comment (nupurbansal) [reply industry_benchmarks/input_structures/prepared_structures/janssen_bace/bace_ciordia_prospective/bace_ciordia_prospective_protein.pdb:1]: Done

File: industry_benchmarks/input_structures/prepared_structures/janssen_bace/bace_p3_arg368_in/bace_p3_arg368_in_ligands.sdf
[the remark this answers is not in the document]
Comment (nupurbansal) [reply industry_benchmarks/input_structures/prepared_structures/janssen_bace/bace_p3_arg368_in/bace_p3_arg368_in_ligands.sdf:1]: Done :)

File: industry_benchmarks/input_structures/prepared_structures/miscellaneous_set/faah/hfaah_prepared.pdb
[the remark this answers is not in the document]
Comment (nupurbansal) [reply industry_benchmarks/input_structures/prepared_structures/miscellaneous_set/faah/hfaah_prepared.pdb:1]: Removed

**Comment by nupurbansal** [comment]
Hi @hannahbaumann Thank you for reviewing. I have made the requested changes. Please re-review whenever convenient for you. Thank you!

**Review by hannahbaumann — APPROVED** [review]
Thanks @nupurbansal , this looks good to me!

**Review by IAlibay — APPROVED** [review]
Thanks @nupurbansal !

**Pull request merged**